adhocracy4/maps/mixins.py

```diff
@@ -47,7 +47,7 @@ def dump_geojson(self):
     def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
         context['mapitems_json'] = self.dump_geojson()
-        context['map_url'] = settings.BASE_MAP
+        context['baseurl'] = settings.MAP_BASEURL
         context['polygon'] = json.dumps(self.module.settings_instance.polygon)
```

Comment:
Is this the URL to the tile server? If yes, I find it confusing to call this base_url.